awx/ui/client/src/home/home.controller.js

```diff
@@ -85,7 +85,7 @@ export default ['$scope','Wait', '$timeout', 'i18n',
                 ProcessErrors($scope, data, status, null, { hdr: i18n._('Error!'), msg: i18n._(`Failed to get dashboard host graph data: ${status}`) });
             });
 
-            Rest.setUrl(GetBasePath("unified_jobs") + "?order_by=-finished&page_size=5&finished__isnull=false&type=workflow_job,job");
+            Rest.setUrl(GetBasePath("unified_jobs") + "?order_by=-finished&page_size=5&finished__isnull=false&type=workflow_job,job&count_disabled=1");
```

Comment:
These queries are *only* pulling back "the last five", so there's need for a full count and paging support.